### ISSUE TITLE
file stopped publishing

### ISSUE THREAD

**Issue by apps4u** [OraOpenSource/apex-nitro#295]
### Issue Description
I got a new mac today and I install this via npm today I checked out my product I re-configured the products now when I run publish i get all the expected out put but not the list of files to upload and no files get uploaded their are no errors on the console.

### APEX Nitro Version
APEX Nitro version 4.1.6

### Operating System
Mac osx 10.14.4

### Node Version
v10.15.3

### Browser
Chrome 73

### APEX Version
19.1

### Web Server
tomcat 7

### Project configuration screenshot
<img width="1573" alt="Screen Shot 2019-04-16 at 5 33 56 pm" src="https://user-images.githubusercontent.com/756522/56190611-ea4b0e80-606d-11e9-83d6-f80575018567.png">
<img width="1528" alt="Screen Shot 2019-04-16 at 5 34 05 pm" src="https://user-images.githubusercontent.com/756522/56190614-eae3a500-606d-11e9-9a98-baea4105e413.png">
<img width="1586" alt="Screen Shot 2019-04-16 at 5 34 14 pm" src="https://user-images.githubusercontent.com/756522/56190616-eae3a500-606d-11e9-9313-aad8bedafdd2.png">

**Comment by vincentmorneau** [comment]
1. What is your Node.js version?
1. Can you also post a screenshot of your terminal after you've done apex-nitro publish?

**Comment by apps4u** [comment]
Node version in description and output from publish command is:

✹  ApexMapping (develop *=) 👽  apex-nitro publish marketing

    _     ____   _____ __  __    _   _  _  _
   / \   |  _ \ | ____|\ \/ /   | \ | |(_)| |_  _ __  ___
  / _ \  | |_) ||  _|   \  /    |  \| || || __|| '__|/ _ \
 / ___ \ |  __/ | |___  /  \    | |\  || || |_ | |  | (_) |
/_/   \_\|_|    |_____|/_/\_\   |_| \_||_| \__||_|   \___/

is compiling marketing ...
...Done!
...Now publishing!
Uploading to 109 - Application Static Files...

SQLcl: Release 19.1 Production on Wed Apr 17 10:50:59 2019

Copyright (c) 1982, 2019, Oracle.  All rights reserved.

Connected to:
Oracle Database 12c Standard Edition Release 12.2.0.1.0 - 64bit Production


SQL> 
Disconnected from Oracle Database 12c Standard Edition Release 12.2.0.1.0 - 64bit Production

Files were uploaded successfully.

**Comment by apps4u** [comment]
Thanks Ive fixed the issue.
I could not get why it did not work. I could use sqlcl from the command line and connect to my remote database.  Anyway it turn out I had the wrong version of java set on my env JAVA_HOME variable. Now java was setup and I could use it but as this relied on reading the JVM home from the env variable It was not working as soon as I noticed that I updated my bash profile with correct version path set for JAVA_HOME.

It only took tow hours of reading wire-shark logs to see that there was some error .
